h2o-core/src/main/java/water/DKV.java

```diff
@@ -185,7 +185,6 @@ static void write_barrier() {
   static public void prefetch( String key_name ) {        get(Key.make(key_name),false); }
 
   static private Value get( Key key, boolean blocking ) {
-    if (key == null) return null;
```

Comment:
this was introduced by POJO commit - we look at that with Tomas, and asked pasha to remove the line.